server/src/vote.rs

```diff
@@ -152,6 +191,21 @@ impl Wrapper<Vec<Vote>> {
 
     Wrapper(votes_with_stake)
   }
+
+  pub fn into_weighted_mep(self, id: &usize, ledger: &Ledger, tip: i64) -> Wrapper<Vec<VoteWithWeight>> {
+    let votes = self.process_mep(&id.to_string(), tip);
```

Comment:
question: is the borrow needed?
clippy is complaining about `needless_borrows_for_generic_args`

Reply:
fixed